src/ui/colortableview.js

```diff
@@ -152,45 +215,69 @@ export default class ColorTableView extends View {
 	 * Creates a static color table grid based on the editor configuration.
 	 *
 	 * @private
+	 * @returns {module:ui/src/colorgrid/colorgridview~ColorGridView}
 	 */
-	createStaticColorTable() {
+	_createStaticColorsGrid() {
 		const colorGrid = new ColorGridView( this.locale, {
 			colorDefinitions: this.colorDefinitions,
 			columns: this.columns
 		} );
 
 		colorGrid.delegate( 'execute' ).to( this );
-		colorGrid.bind( 'selectedColor' ).to( this );
 
 		return colorGrid;
 	}
 
 	/**
-	 * @inheritDoc
+	 * Creates document colors section view and binds it to {@link #documentColors}.
+	 *
+	 * @private
+	 * @returns {module:ui/src/colorgrid/colorgridview~ColorGridView}
 	 */
-	render() {
-		super.render();
+	_createDocumentColorsGrid() {
+		const documentColors = new ColorGridView( this.locale, {
```

Comment:
`ColorGridView` class should not maintain the label. The label should be simply added here in `ColorTableView` as a child before the document colors grid. See https://github.com/ckeditor/ckeditor5-ui/pull/497#issuecomment-493933922

So the label property will no longer be needed